### DISCUSSION TITLE
Web app shows message "Potentially Out of sync" immediately after login

### DISCUSSION THREAD

**Discussion by Matei59** [standardnotes/forum#3119] [General]
**Describe the bug**
As soon as I login into the SN web application (app.standardnotes.com), I see that the bottom right corner contains a yellow text message saying "Potentially out of sync".  This happened 3 times, without any modification to the notes -- the message appears immediately after login. 
I tried also to modify a note and the top right "sync" icon turned green and said that everything was saved. I confirmed the correct saving by logging out and then logging back in and exporting a backup. However, the message "potentially out of sync" is always there. When the warning message is there, the account menu (bottom left) shows also green text that everything is ok (in sync). 

**To Reproduce**
1. Login to the SN web app
2. Observe "out of sync" warning message in bottom right corner

**Expected behavior**
If there is really an out of sync problem, the app should automatically enter into read-only mode, to prevent data loss for the user.
Also, if there is a sync issue, the top icon should not say everything is ok, while bottom message says exactly the opposite.

**Desktop (please complete the following information):**
 - OS: Windows 10 x64
 - Browser: Chrome
 - Version: Version 107.0.5304.88 (Official Build) (64-bit)

**Additional context**
This problem seems to be new, as of today.

**Comment by Matei59** [comment]
![sn_out_of_sync_error](https://user-images.githubusercontent.com/83876893/200573380-7cdf53bb-fcc3-424d-a301-f0862767f75d.jpg)
Please find attached a screenshot with the warning message.

**Comment by effieeee** [comment]
Hi @Matei59, I'm sorry to hear about this. Apart from the "Potentially out of sync" message though, does syncing between your other devices appear to work properly otherwise? 🤔

**Comment by Matei59** [comment]
Hello, thank you for the feedback.  
To answer the question -- I am not using other devices -- I use SN only from the Chrome web browser (from incognito mode).

I just tried to log-in a few minutes ago and I see the same problem as yesterday -- the yellow message appears in bottom right corner (potentially out of sync), the message appears immediately after login, I did not modify any note. 

Please find attached a screenshot of the burger menu (bottom left), it shows everything is ok.
![sn sync status](https://user-images.githubusercontent.com/83876893/200888797-7c2e1903-8460-4444-a482-476acc9f5c2d.jpg)

Of course, I shall stop using SN until this issue is solved, because the possibility of data loss seems to be very high (data not being saved properly to SN server, due to out-of-sync situation). 

I would like to add that my internet connection is very stable, there are no issues there. Also, no changes were done recently to the machine from where SN is being used. 

I look forward to a resolution of this issue.
Thank you very much.

**Comment by effieeee** [comment]
Hi @Matei59 , I see, in this case though we can also try to take a closer look at your account to see what might be causing this. Can I ask you to send us a message at help@standardnotes.com using your Standard Notes account email? I'll also continue assisting you from there, though I won't close this issue on the forum yet either 👍
